### DISCUSSION TITLE
Daily 20/12/2023

### DISCUSSION THREAD

**Discussion by lucasdpt** [Sinan-Karakaya/R-Type#82] [Daily]
J'aide sur le client pour l'implémentation network pour essayer de finir ce soir

**Comment by Sinan-Karakaya** [comment]
Finish implementation of prefabs, and start the development of the MVP

**Comment by JasonLAGOUTE** [comment]
Je fais de la doc sur le network

**Comment by mlbonniec** [comment]
* Implémentation des packets côté client (comme hier)

**Comment by jovanHillion** [comment]
- travail sur le MVP à savoir le jeu
